versions.tf

```diff
@@ -1,10 +1,12 @@
 terraform {
+  required_version = ">= 1.0.0"
   required_providers {
     null = {
       source = "hashicorp/null"
     }
-    metal = {
-      source = "equinix/metal"
+    equinix = {
+      source = "equinix/equinix"
+      version = "1.14.1"
```

Comment:
We could use `~> 1.14` here so that it's easier to pull in future minor/patch versions in the `v1` series (which _should_ be backwards-compatible).

Reply:
updated